pkg/api/types.go

```diff
@@ -2187,6 +2189,8 @@ type OwnerReference struct {
 	// UID of the referent.
 	// More info: http://releases.k8s.io/HEAD/docs/user-guide/identifiers.md#uids
 	UID types.UID `json:"uid"`
+	// If set to true this reference points to managing controller
```

Comment:
If true, this reference points to the managing controller, and this reference must be first in the list if there are multiple ownerReferences.